File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/Images_View.cpp

#include "Images_View.h"
#include "Images_View.h"
#include <string>

namespace widgetassertions
{
  std::string Images_View::getStatusImageName( ) 
  {
    return this->StatusImageName;
  }
  bool Images_View::getIsStatusImageEnabled( ) 
  {
    return this->isStatusImageEnabled;
  }
  bool Images_View::getIsStatusImageVisible( ) 
  {
    return this->isStatusImageVisible;
  }
  std::string Images_View::getUpOrDownImageName( ) 
  {
    return this->UpOrDownImageName;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/Images_View.cs
/// <filename>
///     Images_View.cs
/// </filename>


namespace widgetassertions
{
    public class Images_View
  {
        private string StatusImageName;
     
        public     virtual string getStatusImageName()
    {
      return this.StatusImageName;
    }
     
        private bool isStatusImageEnabled;
     
        public     virtual bool getIsStatusImageEnabled()
    {
      return this.isStatusImageEnabled;
    }
     
        private bool isStatusImageVisible;
     
        public     virtual bool getIsStatusImageVisible()
    {
      return this.isStatusImageVisible;
    }
     
        private string UpOrDownImageName;
     
        public     virtual string getUpOrDownImageName()
    {
      return this.UpOrDownImageName;
    }
  }
}

